MOCC/src/configurations/NewPyTango/device_proxy.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+
+class DeviceProxy:
+    def __init__(self, resource_path_URI):
+        self.resource_path_URI = resource_path_URI
+        self.bind_functions()
+
+    def test_connection(self):
+        return 'SUCCESS'
+
+    # Getting the functions of the device server
+    def get_function_list(self):
+        uri = 'http://127.0.0.1:5000' + self.resource_path_URI + '/functions'
```

Comment:
should I replace resource_path_URI with just a name, like : 'health_monitor' and then internally get the resource_path_URI through a database?